src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int colorsCount = 3;
```

Comment:
what if Color enum would contain, lets say, 1 value, and Random would give us 2nd index? what exception would be thrown from the getRandomColor() method? try to fix this, by using Color.values().length instead of colorsCount contant.